hugo/content/en/docs/howto/constrain-list-ints-no-duplicates/index.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: How to constrain a list of integers to have no duplicates
-weight: 
+weight:
 draft: false
----
\ No newline at end of file
+coming_soon: true
```

Comment:
I'm in doubt if we should call this coming_soon because it could be pages are disabled for another reason (doc outdated or something). Maybe just name this 'disabled' ?

Reply:
@Lyoness What do you think is best content-wise? 'disabled' or 'coming_soon'. What's more clear for you?

Reply:
@JorindeUsMedia yes, `disabled` is good.